add-on/preferences/prefs.js

```diff
@@ -1,4 +1,5 @@
 pref("loop.enabled", true);
+pref("loop.remote.autostart", true);
```

Comment:
I'd suggest we land this as false initially, since it'll take a day or two to get the social patches into nightly & uplifted.